salt/_modules/metalk8s_network.py

```diff
@@ -68,3 +69,13 @@ def get_cluster_dns_ip():
     range.
     '''
     return _pick_nth_service_ip(COREDNS_ADDRESS_NUMBER)
+
+
+def get_oidc_service_ip():
```

Comment:
This is used in the rendered chart and chart values, so either in the same commit, or before it.

Reply:
The commit message should remove the mention about Ingress not being a valid approach, as that's what we use in the end.